### PR TITLE
security account_move_template: use safe_eval

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

Fast-tracking it.

**Comment by gurneyalex** [comment]
@pedrobaeza hmm not sure it does not break anything...

**Comment by elicoidal** [comment]
Agree: I understand we need to act quickly but should not remove basic review.

**Comment by pedrobaeza** [comment]
Well, why it would break something? It's only a replace of one expression by another. The only break it can happen is if the expression is not safe, and this is what we intend.

**Comment by eLBati** [comment]
Travis is complaining

```
2016-02-10 07:32:39,823 23546 CRITICAL openerp_test openerp.modules.module: Couldn't load module account_move_template
2016-02-10 07:32:39,823 23546 CRITICAL openerp_test openerp.modules.module: No module named safe_eval
```

I did not test locally

File: account_move_template/account_document_template.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
 from openerp.osv import fields, orm
 from openerp.tools.translate import _
+from openerp.tools.translate.safe_eval import safe_eval
```
Comment (tafaRU) [inline account_move_template/account_document_template.py:24]: This should be replaced by `from openerp.tools.safe_eval import safe_eval`

**Comment by tafaRU** [comment]
@gurneyalex @pedrobaeza I'm preparing a PR to fix https://github.com/OCA/account-financial-tools/pull/350#issuecomment-182758077

**Comment by tafaRU** [comment]
I see it's already been fixed in 7.0 branch. Okay, never mind!

**Pull request merged**